pywhat/Data/regex.json

```diff
@@ -1,5 +1,31 @@
 [
-    {
+   {
+      "Name": "PGP Public Key",
+      "Regex": "/^(-----BEGIN PGP PUBLIC KEY BLOCK-----).*([a-zA-Z0-9//\n\/\\.\\:\\+\\ \\=]+).*(-----END PGP PUBLIC KEY BLOCK-----)$",
+      "plural_name": false,
+      "Description": null,
+      "Rarity": 1,
+      "URL": "",

```

Comment:
Thank you for quickly fixing everything! Can you instead change both of these to `null`? After this, I think your PR can get merged. 😄

Reply:
Thank you for allowing me to contribute. I just fixed it and changed both to `null`